frontend/src/model/schema/Column.ts

```diff
@@ -77,7 +79,17 @@ export default class Column implements BasicColumn {
   }
 
   public get nullable() {
-    return this.sourceColumn.nullable;
+    return this.nullableConstraint('maximal');
+  }
+
+  public nullableConstraint(constraintPolicy: string) {
```

Comment:
Würde das mit Tooltips im toggle dafü rim UI erklären. Und vielleicht (optional) würde sich statt string eigener typ empfehlen, sowas wie

```typescript
type ConstraintPolicy = 'minimal' | 'schema' | 'maximal';
```

Hat sonst Vertipppotenzial mit den ganzen magic strings, auch in den anderen Dateien